### PR TITLE
update summer 2021's proposals

### PR DESCRIPTION
**Pull request by shenhonglei**
Signed-off-by: honglei <hongleishen@hotmail.com>

### CODE REVIEW

File: sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
   
     https://github.com/kubesphere/community/tree/master/sig-advocacy-and-outreach/summer-ospp
   
-- [ ] Customize Notification Receivers Using a Webhook @火冰界 
+- [x] ~~Customize Notification Receivers Using a Webhook @火冰界~~
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md:28]: `~~`??

File: sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md [hunk 2]
```diff
@@ -14,7 +14,7 @@
   
     https://github.com/kubesphere/community/tree/master/sig-advocacy-and-outreach/summer-ospp
   
-- [ ] KubeSphere Tekton Integration @shaowen
+- [x] KubeSphere Tekton Integration @shaowen
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md:17]: The GitHub `shaowen` is incorrect.

File: sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 > 1. 可以直接在 GitHub 上提交完善信息
 > 2. 可以直接在 HackMD 上补充完善信息，由管理员统一提交至 GitHub.
 
-- [ ] KubeSphere Jenkins Client Refactor @Rick 
+- [x] KubeSphere Jenkins Client Refactor @Rick
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md:6]: The GitHub ID should be [linuxsuren](https://github.com/linuxsuren) instead of `@Rick`

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md (hunk 1 above)
Comment (shenhonglei) [reply sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md:28]: This is the nickname of WeChat

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**